csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md

```diff
@@ -114,24 +113,29 @@ That’s why the replies sometimes use part of what you typed.
 
 The quality of these rules determines how intelligent the chatterbot will sound.  
 
-While Alice is undoubtedly more intelligent than Eliza, Alice still has her limitations. Sometimes she even admits to those limitations!
+While Alice is undoubtedly more intelligent sounding than Eliza, Alice still has her limitations.
+Sometimes she even admits to those limitations!
 
-**Judge**: What will happen in the year 2000  
+**Judge**: What will happen in the year 2000?
```

Comment:
Need to add back in the new line
![image](https://user-images.githubusercontent.com/16066822/56937744-838c2180-6b52-11e9-9f77-2592f4aad52e.png)

Reply:
Not entirely sure what that character is, but I copy-pasted it back in